### ISSUE TITLE
The shortcut key prompt for opening the browser when using wsl2 is redundant

### ISSUE THREAD

**Issue by aiktb** [wxt-dev/wxt#1262]
### Describe the bug
cc @nishu-murmu

![image](https://github.com/user-attachments/assets/ac62f42b-4579-4030-acc1-640838781ca1)
I noticed there is already an isWsl utility function, but I'm not sure how to add it. The following modification directly removes the shortcut registration step, but perhaps more shortcuts that can be used in wsl2 will be added in the future.
![image](https://github.com/user-attachments/assets/c9b0d512-dabe-4824-9648-dbbdd6860576)

### Reproduction

Using wsl2 then pnpm dev

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 24.04.1 LTS 24.04.1 LTS (Noble Numbat)
    CPU: (18) x64 Intel(R) Core(TM) Ultra 5 125H
    Memory: 20.06 GB / 23.47 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh
  Binaries:
    Node: 22.11.0 - ~/.nvm/versions/node/v22.11.0/bin/node
    Yarn: 1.22.22 - /mnt/c/Program Files/nodejs/yarn
    npm: 10.9.0 - ~/.nvm/versions/node/v22.11.0/bin/npm
    pnpm: 9.15.0 - /mnt/c/Program Files/nodejs/pnpm
    bun: 1.1.10 - ~/.bun/bin/bun
```

### Used Package Manager

pnpm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://wxt.dev/guide/installation.html).
- [X] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by aklinker1** [comment]
Ahh yes, this was introduced by one of my changes on the PR. I'll handle this

**Comment by aklinker1** [comment]
Fix released in [`v0.19.20`](https://github.com/wxt-dev/wxt/releases/tag/wxt-v0.19.20)
